### PR TITLE
VIPCS Flagged escaping fixes.

### PR DESCRIPTION
**Pull request by rinatkhaziev**

### CODE REVIEW

File: classes/class-woothemes-sensei-certificates.php [hunk 1]
```diff
@@ -1062,7 +1062,7 @@ public function certificates_user_settings_form( $user ) {
...
 		            <p>
-		            	 <input type="checkbox" value="yes" name="certificates_user_public_view" <?php checked( $view_setting, true ); ?>/> <?php _e( 'Allow my Certificates to be publicly viewed', 'sensei-certificates' ); ?> <input type="submit" name="certificates_user_meta_save" class="certificates-submit complete" value="<?php echo apply_filters( 'sensei_certificates_save_meta_button', __( 'Save', 'sensei-certificates' ) ); ?>"/>
+		            	 <input type="checkbox" value="yes" name="certificates_user_public_view" <?php checked( $view_setting, true ); ?>/> <?php esc_html_( 'Allow my Certificates to be publicly viewed', 'sensei-certificates' ); ?> <input type="submit" name="certificates_user_meta_save" class="certificates-submit complete" value="<?php echo esc_attr( apply_filters( 'sensei_certificates_save_meta_button', __( 'Save', 'sensei-certificates' ) ) ); ?>"/>
```
Comment (donnapep) [inline classes/class-woothemes-sensei-certificates.php]: `esc_html_` => `esc_html_e`

**Comment by donnapep** [comment]
@rinatkhaziev Would you mind rebasing against `master`? Thx!

**Comment by rinatkhaziev** [comment]
@donnapep sure, this should be done now.

File: classes/class-woothemes-sensei-certificates.php (hunk 1 above)
Comment (rinatkhaziev) [reply classes/class-woothemes-sensei-certificates.php]: Thanks for the catch, should be fixed now.

**Pull request merged**